main_routers/system_router.py

```
@@ -0,0 +1,858 @@
+            '.webp': 'image/webp'
+        }.get(ext, 'application/octet-stream')
+        
+        # 返回图片数据
+        return Response(content=image_data, media_type=mime_type)
+    except Exception as e:
+        logger.error(f"代理图片访问失败: {str(e)}")
+        return JSONResponse(content={"success": False, "error": f"访问图片失败: {str(e)}"}, status_code=500)
+
+@router.post('/proactive_chat')
+async def proactive_chat(request: Request):
+    """主动搭话：根据概率选择使用图片或热门内容，让AI决定是否主动发起对话"""
```

Ruff (0.14.8)

Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)